src/main/java/pl/piekoszek/gorskimatches/token/AccountController.java

```diff
@@ -1,18 +1,29 @@
 package pl.piekoszek.gorskimatches.token;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
+import pl.piekoszek.gorskimatches.repository.AccountRepository;
 
+@CrossOrigin
 @RestController
 @RequestMapping("auth")
 class AccountController {
+    @Autowired
+    AccountRepository accountRepository;
+
     private EmailService emailService;
 
-    AccountController(EmailService emailService){
+    AccountController(EmailService emailService) {
         this.emailService = emailService;
     }
 
     @PostMapping("email")
     void getEmail(@RequestBody AccountInfo accountInfo) {
         emailService.sendRegistrationLink(accountInfo.getEmail());
```

Comment:
hmmm it's not only registration, but login as well

Reply:
ok